qcfractal/storage_sockets/mongo_socket.py

```diff
@@ -115,52 +117,71 @@ def __init__(self,
         else:
             self.client = pymongo.MongoClient(url, port)
 
+        try:
+            version_array = self.client.server_info()['versionArray']
+            if version_array[0] * 10 + version_array[1] < 32:
```

Comment:
`tuple(client.server_info()['versionArray']) > (3, 2, 0, 0)`?

Reply:
Yeah, that is probably the correct way to handle that.